src/components/ImageView/index.spec.tsx

```diff
@@ -34,25 +36,74 @@ describe(__filename, () => {
     );
   });
 
-  it('does not render an img tag when btoa fails', () => {
+  it('throws an error when btoa fails', () => {
+    const message = 'An error message';
     const _btoa = jest.fn().mockImplementation(() => {
-      throw new Error();
+      throw new Error(message);
     });
 
-    const root = render({ _btoa });
-
-    expect(root.find('img')).toHaveLength(0);
+    expect(() => {
+      render({ _btoa });
+    }).toThrow(message);
   });
 
-  it('logs a debug message when btoa fails', () => {
-    const _btoa = jest.fn().mockImplementation(() => {
-      throw new Error();
-    });
+  it('logs an error message if an image has an invalid mimeType', () => {
     const _log = createFakeLogger();
 
-    render({ _btoa, _log });
+    render({ _log, mimeType: 'invalid/type' });
+
+    expect(_log.error).toHaveBeenCalled();
+  });
+
+  it('renders a message if an image has an invalid mimeType', () => {
+    const root = render({ mimeType: 'invalid/type' });
+
+    expect(root.find('p')).toHaveText('Unrecognized image format');
+  });
+
+  it('calls sanitize on svg files', () => {
+    const _sanitize = jest.fn();
+    const content = 'some content';
+    const mimeType = 'image/svg+xml';
+    render({ _sanitize, content, mimeType });
+
+    expect(_sanitize).toHaveBeenCalledWith(content);
+  });
+
+  it('recognizes upper cased svg mimeType', () => {
+    const _sanitize = jest.fn();
+    const content = 'some content';
+    const mimeType = 'IMAGE/SVG+XML';
+    render({ _sanitize, content, mimeType });
+
+    expect(_sanitize).toHaveBeenCalledWith(content);
+  });
+
+  it('does not call sanitize on non-svg files', () => {
+    const _sanitize = jest.fn();
+    const content = 'some content';
+    const mimeType = 'image/png';
+    render({ _sanitize, content, mimeType });
+
+    expect(_sanitize).not.toHaveBeenCalled();
+  });
+
+  it('sanitizes an svg file', () => {
+    const content = `
+      <svg version="1.1">
+        <circle fill="red" />
+        <script>alert('XSS via SVG')</script>
+        </svg>
```

Comment:
My brain really wants this to be indented two spaces to the left but no biggie 😆